packages/electron-app/src/main/index.ts

```diff
@@ -7,9 +7,9 @@ import path from 'path';
 import Pino from 'pino';
 import url from 'url';
 
-import { CSP, isSubstrateRunning, killSubstrate, runSubstrateDev, staticPath } from './util';
+import { isSubstrateRunning, killSubstrate, runSubstrateDev, staticPath } from './util';
 
-const { app, BrowserWindow, session } = electron;
+const { app, BrowserWindow } = electron;
```

Comment:
nit: move absolute import to top of file.

Reply:
This is not an absolute import. Unless you mean change it to `import { app, BrowserWindow } from 'electron'`